src/olympia/devhub/views.py

```diff
@@ -1574,7 +1574,17 @@ def _submit_upload(
         if existing_properties
         else []
     )
-    submit_notification_warning = get_config('submit_notification_warning')
+    flag = waffle.get_waffle_flag_model().get('enable-submissions')
+    warning = (
+        'Add-on uploads are temporarily unavailable' + ': ' + flag.note
+        if hasattr(flag, 'note')
+        else '.'
+    )
```

Comment:
Can you add a test to verify what's rendered when there is no note set, but the flag is enabled?  I missed that case before.